src/hooks/useENSPendingRegistrations.tsx

```diff
@@ -1,13 +1,17 @@
 import { useCallback, useMemo } from 'react';
-import { useSelector } from 'react-redux';
+import { useDispatch, useSelector } from 'react-redux';
 import { useAccountSettings, useENSRegistration } from '.';
 import { ENSRegistrationState } from '@rainbow-me/entities';
+import { removeExpiredRegistrations } from '@rainbow-me/redux/ensRegistration';
 import { AppState } from '@rainbow-me/redux/store';
 import getENSNFTAvatarUrl from '@rainbow-me/utils/getENSNFTAvatarUrl';
 
 export default function useENSPendingRegistrations() {
   const { accountAddress } = useAccountSettings();
   const { removeRegistrationByName } = useENSRegistration();
+  const dispatch = useDispatch();
+
+  dispatch(removeExpiredRegistrations(accountAddress));
```

Comment:
could you move this one to its own method and export it from the hook? this pattern feels strange to me, that way we can call the new method from a component when we want instead of when the hook is mounted